quality_control/demo/quality_control_demo.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data noupdate="1">
+        <record model="qc.test" id="qc_test_1">
+            <field name="name">Generic Test (demo)</field>
+            <field name="type">generic</field>
+            <field name="active" eval="True" />
+            <field name="category" ref="qc_test_template_category_generic" />
+        </record>
+
+        <record model="qc.test.question" id="qc_test_question_1">
+            <field name="name">Question 1</field>
```

Comment:
Yes, I told you that I didn't have clear how to do a demo test, because we will have to put one which fits demo data... this is a sample just to have something...